### DISCUSSION TITLE
Captcha Support

### DISCUSSION THREAD

**Discussion by jeremiah1232** [titaniumnetwork-dev/Vanadium#9] [General]
hcaptch dosent show up |   not on discord not on youtube or any website

**Comment by Null-n-V0id** [comment]
Unfortunately this is a problem with the Corrosion backend. I'm considering dropping Corrosion in the near future, which should solve this problem.

**Comment by peacocksir** [comment]
So are yall gonna drop Corrosion? If so when bc captch's don't even work for me :(

+ Corrosion is also broke[i think(for me at least)]

**Comment by Null-n-V0id** [comment]
> So are yall gonna drop Corrosion? If so when bc captch's don't even work for me :(
> 
>     * Corrosion is also broke[i think(for me at least)]

Yes. I'm only waiting for an in-progress alternative to release, and then I'll move Vanadium over.

**Comment by Null-n-V0id** [comment]
Still an ongoing problem. Will keep you posted.
